win32ss/drivers/miniport/bochs/bochsmp.c

```diff
@@ -0,0 +1,744 @@
+/*
+ * PROJECT:     ReactOS Bochs graphics card driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Bochs graphics card driver
+ * COPYRIGHT:   Copyright 2022 Hervé Poussineau <hpoussin@reactos.org>
+ */
+
+#include "bochsmp.h"
+
+static const BOCHS_SIZE BochsAvailableResolutions[] = {
+    { 640, 480 },   // VGA
+    { 800, 600 },   // SVGA
+    { 1024, 600 },  // WSVGA
+    { 1024, 768 },  // XGA
+    { 1152, 864 },  // XGA+
+    { 1280, 720 },  // WXGA-H
+    { 1280, 768 },  // WXGA
+    { 1280, 960 },  // SXGA-
+    { 1280, 1024 }, // SXGA
+    { 1368, 768 },  // HD ready
+    { 1400, 1050 }, // SXGA+
+    { 1440, 900 },  // WSXGA
+    { 1600, 900 },  // HD+
+    { 1600, 1200 }, // UXGA
+    { 1680, 1050 }, // WSXGA+
+    { 1920, 1080 }, // FHD
+    { 2048, 1536 }, // QXGA
+    { 2560, 1440 }, // WQHD
+    { 2560, 1600 }, // WQXGA
+    { 2560, 2048 }, // QSXGA
+    { 2800, 2100 }, // QSXGA+
+    { 3200, 2400 }, // QUXGA
+    { 3840, 2160 }, // 4K UHD-1
+};
+
+static VOID
```

Comment:
Can you please add a `CODE_SEG("PAGE")` and `PAGED_CODE()` for every function except the `DriverEntry` routine?

Reply:
OK for `CODE_SEG("PAGE")`.
However, `PAGED_CODE()` depends of `RtlAssert`, which is not exported by videoprt.